src/pyats/contrib/creators/netbox.py

```diff
@@ -368,11 +642,20 @@ def _generate(self):
                     current.setdefault("alias", "{}_{}"
                                             .format(device_name, interface_name))
                     self._set_value_if_exists(current, "type", 
-                                        self._format_type(interface_name.lower()))
+                                        self._format_type(
+                                            interface_name.lower(), 
+                                            interface["type"]
+                                        ))
                     if current.get('type') is None:
                         logger.info(f"{device_name} interface {interface_name.lower()} is not valid, skipping")
                         del interfaces[interface_name]
                         continue
+                    
+                    # Use the cable information from Netbox to configure link on interface
+                    self._set_value_if_exists(
+                        current, "link", 
+                        self._get_info(interface, ["cable", "id"], lambda link: f"cable_num_{link}")
```

Comment:
please change to format() from f-strings for places where applicable. We are still not using f-string because we still support python3.5. f-strings is supported after python3.6+. (we are planning to start using f-string from Jan 2021)

Reply:
@tahigash I have updated the code to remove all use of f strings from the code. There were a few other uses that were in place before my changes but I pulled them all out and replaced with .format().